### PR TITLE
feat: enhance tooltip to accept a custom wrapper

### PR DESCRIPTION
**Pull request by georgesimos**
### Background

The tooltip component is a bit limited, so I introduced the content wrapper that will enhance the tooltip capability to modify the content container, e.g. background colour or adding an arrow!
Screenshot 2022-12-02 at 2 38 34 PM

### Changes

- Add a contentWrapper property
- Keep the existing as a default

### Testing

- Testing locally

### CODE REVIEW

File: src/components/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -7,20 +7,30 @@ import Box from '../Box';
...
+  );
+};
+
 export interface TooltipProps {
-  /** The string or HTML that the tooltip will show*/
+  /** The string or HTML that the tooltip will show */
   content: string | React.ReactElement;
 
+  /** The wrapper that contains the contnet */
+  contentWrapper?: React.ElementType;
```
Comment (3nvi) [inline src/components/Tooltip/Tooltip.tsx:23]: optional: I'd rename that to just `wrapper`

**Review by 3nvi — COMMENTED** [review]
@georgesimos It looks fine, but before I approve it, can we add a test around this new capability?

File: src/components/Tooltip/Tooltip.tsx (hunk 1 above)
Comment (georgesimos) [reply src/components/Tooltip/Tooltip.tsx]: Done!

**Review by kouknick — APPROVED** [review]
LGTM!

**Pull request merged**